### PR TITLE
New Term: Pandas `.fillna()`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Method from the Target Keyword list I missed with the Pandas PR

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Just a few comments/suggestion from me for this one. 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestions.

File: content/pandas/concepts/dataframe/terms/fillna/fillna.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+|    `value`     | scalar, dict, Series, Dataframe         | Value used to fill holes. A scalar or a dict/Series/DataFrame specifying replacement values for each column.       |
+|    `method`    | 'backfill','bfill','pad','ffill',`None` | 'backfill'/'bfill' fills holes with next valid observation. 'pad'/'ffill' fills holes with last valid observation. |
+|     `axis`     | 0/1 or 'index'/'columns'                | Axis along which to fill missing values.                                                                           |
+|   `inplace`    | bool                                    | If `True`, alters the existing `DataFrame` rather than returning a new one. Defaults to `False`.                   |
+|    `limit`     | int                                     | Maximum consecutive items to back/forward fill. Defaults to `None`.                                                |
+
+## Example
+
+In the following example, the `.fillna()` method is used in to fill `NA` values in a DataFrame first with a scalar, then with a dict:
```
Comment (KTom101) [inline content/pandas/concepts/dataframe/terms/fillna/fillna.md:38]:
```suggestion
In the following example, the `.fillna()` method is used to fill in `NA` values in a DataFrame first with a scalar, then with a dict:
```

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski This looks pretty much ready to merge, in my opinion. I just added a couple minor things.

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

**Review by Dusch4593 — APPROVED** [review]

**Review by KTom101 — APPROVED** [review]
@SSwiniarski This looks like it's ready to merge. 
@Dusch4593 How does it look to you?

**Pull request merged**